drivers/xen/console/Config.uk

```diff
@@ -1,7 +1,6 @@
 menuconfig LIBXEN_CONSOLE
 	bool "Hypervisor console"
 	select LIBUKCONSOLE
-	select LIBNOLIBC
```

Comment:
What happens we put here?
`select LIBNOLIBC if !HAVE_LIBC`
This is probably more accurate.

Reply:
That's what I was about to say as well. Libraries should explicitly declare their dependencies rather than relying on implicit selection by other libraries.

Reply:
Lemme try. That should work. I wasn't aware with the best practice of declare your dependencies, but it makes sense.

Reply:
@skuenzer , @michpappas , it works OK. I update the commit, commit description and PR description to reflect this.

Reply:
Thanks!